onnxscript/function_libs/torch_lib/ops/linalg.py

```diff
@@ -44,10 +44,28 @@ def aten_linalg_cond(self: TensorType, p: Optional[float] = None) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_linalg_cross(self: TensorType, other: TensorType, dim: int = -1) -> TensorType:
-    """linalg_cross(Tensor self, Tensor other, *, int dim=-1) -> Tensor"""
+@torch_op("aten::linalg_cross")
+def aten_linalg_cross(self: TTensor, other: TTensor, dim: int = -1) -> TTensor:
```

Comment:
We can just register it under https://github.com/microsoft/onnxscript/blob/bcbf9f70afa6526478a57fe405ad464c4adfb631/onnxscript/function_libs/torch_lib/ops/core.py#L2240

Reply:
I don’t think so. They are the exact implementation. One is alias of another

Reply:
Unless they are implemented differently?